src/pages/cli/project/troubleshooting.mdx

```diff
@@ -209,6 +209,29 @@ If `amplify push` fails while deploying any of the scenarios below, refer to the
 1.	[Backfill OpenSearch index from DynamoDB table](/cli/graphql/troubleshooting/#deploying-multiple-index-changes-at-once)
 1.	[Index with Multi-Sort Key Fields](/cli/graphql/troubleshooting/#index-with-multiple-sort-key-fields)
 
+## Troubleshooting "amplify pull" failures
+
+### Pull failures with large number of UI components
+<Callout information>
+
+**Errors:**
+
+"⠋ Generating UI components...🛑 Codegen job never succeeded before timeout"
+"✖ Failed to sync UI components"
+</Callout>
+
+<!-- cSpell:disable -->
```

Comment:
what is the reason for disabling cspell?

Reply:
I got a spellcheck error for `CODEGENJOB` as follows -
```
> git -c user.useConfigOnly=true commit --quiet --allow-empty-message --file -
yarn run v1.22.19
$ git diff --name-only --cached | awk "/src.*\.mdx/{print}" | npx cspell --no-must-find-files --file-list stdin

1/1 ./src/pages/cli/project/troubleshooting.mdx 747.59ms X
/Documents/amplify-playground/docs/src/pages/cli/project/troubleshooting.mdx:225:222 - Unknown word (CODEGENJOB)
/Documents/amplify-playground/docs/src/pages/cli/project/troubleshooting.mdx:229:19 - Unknown word (CODEGENJOB)
CSpell: Files checked: 1, Issues found: 2 in 1 files
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
husky - pre-commit hook exited with code 1 (error)
```

Reply:
Ah can you add that word to the dictionary to pass the spellcheck? https://github.com/aws-amplify/docs/blob/main/cspell.json#L4

Reply:
added `CODEGENJOB` to cspell.json